DOCS/man/input.rst

```diff
@@ -2456,7 +2480,8 @@ Property list
     ways. The property is unavailable if no video is active.
 
 ``focused``
-    Whether the window has focus. Currently works only on X11 and Wayland.
+    Whether the window has focus. Currently works only on X11, Wayland and
+    MacOS.
```

Comment:
please don't introduce another variant of how the apple os is written in the man, just use macOS to be somewhat consistent. see a bit below.

Reply:
Fixed

Reply:
it's still off in the commit message.

Reply:
Fixed that too